flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -553,6 +553,33 @@ def set_python_executable(self, python_exec: str):
             .getEnvironmentConfig(self._j_stream_execution_environment)
         env_config.setString(jvm.PythonOptions.PYTHON_EXECUTABLE.key(), python_exec)
 
+    def add_jars(self, jars_path: str):
```

Comment:
Do you think it makes sense to change the interface a bit as following? 
```suggestion
    def add_jars(self, *jars: str):
```

Besides, we should also consider cases where users may call add_jars multiple times.